src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -668,9 +668,9 @@ public class LibraryServiceTest {
     List<BookName> names = Arrays.asList(namesElement);
     List<ShelfName> shelves = new ArrayList<>();
 
-    PagedListResponse<FindRelatedBooksRequest, FindRelatedBooksResponse, BookName> pagedListResponse = api.findRelatedBooks(names, shelves);
+    FindRelatedBooksPagedResponse pagedListResponse = api.findRelatedBooks(names, shelves);
 
-    List<BookName> resources = Lists.newArrayList(pagedListResponse.iterateAllElements());
+    List<String> resources = Lists.newArrayList(pagedListResponse.iterateAllElements());
```

Comment:
iterateAllElements() now always iterates over the basic type (strings), and FindRelatedBooksPagedResponse has an additional method iterateAllAsBookName(). I will add a test for this method as well